docs/source/release_notes/release-notes.rst

```diff
@@ -86,6 +102,12 @@ develop
          - Fixed a bug in LockServiceImpl (caused by a bug in AbstractQueuedSynchronizer) where a race condition could cause a lock to become stuck indefinitely.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2799>`__)
 
+    *    - |devbreak|
+         - Deleted the TTL duration field from the ``Cell`` class.
+           The interface ``ExpiringKeyValueService`` and implementations ``CassandraExpiringKeyValueService`` and ``CqlExpiringKeyValueService`` have also been removed.
+           Additionally, ``StreamTableDefinitionBuilder.expirationStrategy`` has been removed.
```

Comment:
would it make sense to explain that these were never used, or something along those lines?